src/Psalm/Internal/PluginManager/ComposerLock.php

```diff
@@ -30,12 +30,9 @@ public function __construct(array $file_names)
      */
     public function isPlugin($package): bool
     {
-        return is_array($package)
-            && isset($package['name'])
+        return isset($package['name'], $package['extra']['psalm']['pluginClass'])
```

Comment:
I would probably put that after `is_array($package)` check.

Reply:
👍